.flake8

```diff
@@ -36,6 +36,8 @@ ignore =
     S110,
     # ignore multiple `##` starting a comment
     E266,
+    # ignore "unused import" errors, already handled by pyright
+    F401
```

Comment:
I'd rather we don't make this change, and at least not in this pull. Other things are covered by both flake8 and pyright, and we've been known to find pyright bugs, so this should stay enabled.

Reply:
This is due to a flake8 bug <sup>:^)</sup>, the alternative would be adding several inline `noqa`s, which wouldn't be great either.

```py
from datetime import timedelta
from typing import Generic, TypeVar

T = TypeVar("T")


class A(Generic[T]):
    ...


class B(A["timedelta"]):
    ...
```

```py
$ flake8 test.py
test.py:1:1: F401 'datetime.timedelta' imported but unused
```